simpleline/render/io_manager.py

```diff
@@ -298,39 +298,39 @@ def _process_input(self, active_screen, key):
         try:
             key = active_screen.ui_screen.input(active_screen.args, key)
             if key == InputState.PROCESSED:
-                return UserInputResult.PROCESSED
+                return UserInputReaction.NOOP
             elif key == InputState.PROCESSED_AND_REDRAW:
-                return UserInputResult.REFRESH
+                return UserInputReaction.REDRAW
```

Comment:
What about `UserInputAction?` 

Btw. do we use it in anaconda? Does it mean, that we will have to change TUI again?

Reply:
Nice idea, I'll change that.

This is used only internally so it shouldn't require any change in Anaconda.
In case of the new `PROCESS_AND_*`  they should be backward compatible so it also shouldn't cause any issues in Anaconda.